mmdet3d/datasets/pipelines/loading.py

```diff
@@ -149,7 +149,7 @@ def _load_points(self, pts_filename):
             points = np.frombuffer(pts_bytes, dtype=np.float32)
         except ConnectionError:
             mmcv.check_file_exist(pts_filename)
-            if pts_filename.endswith('.npy'):
+            if pts_filename.endswith('.npy') or pts_filename.endswith('.npz'):
```

Comment:
The loaded `npz` file is something like a dict instead of a np.ndarray? I think the following transform can't operate correctly on that?